src/commands/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+mod args;
+mod cd;
+mod cp_mv;
+mod exit;
+mod mkdir;
+mod rm;
+mod sleep;
+
+pub use cd::*;
+pub use cp_mv::*;
+pub use exit::*;
+pub use mkdir::*;
+pub use rm::*;
+pub use sleep::*;
```

Comment:
Is this really necessary?

Reply:
Do you mean not having these statements? I just tried it without it, but then it becomes really verbose.

Or do you mean make it more explicit and remove the `*`? I think it's ok to just re-export these modules.

Reply:
I think reexporting modules would be preferable to `*`

Reply:
I think it exposes private details about the module structure in here.

![image](https://user-images.githubusercontent.com/1609021/157728933-05a0556f-c1c9-4c92-b3ea-0cce8ecc9e1e.png)

Reply:
Okay, sure. Let's leave it as is for now